simulator/src/framebuffer.rs

```diff
@@ -116,7 +113,7 @@ impl DrawTarget<Rgb888> for Framebuffer {
             let color = &[fill_color.r(), fill_color.g(), fill_color.b()];
 
             let tl = item.primitive.top_left;
-            let br = item.primitive.bottom_right;
+            let br = item.primitive.bottom_right();
```

Comment:
Shouldn't these be either both method calls or both property accesses?

Reply:
They probably should at some point, but these methods are still work in progress, as mentioned in the comment in `Rectangle::bottom_right`.